### ISSUE TITLE
Error on Selenium in Docker

### ISSUE THREAD

**Issue by NeoUKR** [Pullger/pullgerSquirrel#3]
Error on Selenium in Docker:

```
session deleted because of page crash
from unknown error: cannot determine loading status
```

**Comment by NeoUKR** [comment]
**stack overflow:** https://stackoverflow.com/questions/53902507/unknown-error-session-deleted-because-of-page-crash-from-unknown-error-cannot

**Solution**: appended chrome option:  
chrome_options.add_argument('--disable-dev-shm-usage')

**Comment by NeoUKR** [comment]
Fixed in release 0.2.3.1
